src/api/c/topk.cpp

```diff
@@ -0,0 +1,88 @@
+/*******************************************************
+ * Copyright (c) 2018, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#include <af/statistics.h>
+#include <af/data.h>
+
+#include <handle.hpp>
+#include <topk.hpp>
+#include <common/ArrayInfo.hpp>
+#include <common/err_common.hpp>
+#include <backend.hpp>
+
+using namespace detail;
+
+template<typename T>
+af_err topkWithIndices(af_array *v, af_array* i, const af_array in,
```

Comment:
can you change this function name to be topk as well since we are not explicitly saying with indices anywhere now.